### ISSUE TITLE
Channel and Devices

### ISSUE THREAD

**Issue by shroffk** [ControlSystemStudio/cs-studio#1039]
- Add the ability to add/remove/modify properties associated with channels.
- Add adapter for converting a PV(s) to channel(s) (the reverse already exists)
- Create a device interface

**Comment by berryma4** [comment]
Yes!  Can channel be made an interface?  So others that don't use channelfinder can us it?

**Comment by berryma4** [comment]
Commenting on the channelviewer additions.  It seems odd that everything accepts a multi-select except "Modify Channel".
